liquid_node/commands.py

```diff
@@ -1,4 +1,5 @@
 from pathlib import Path
+import collections
```

Comment:
Please write this as `from collections import defaultdict`, we already have the concept of collections, so this would avoid confusion.